flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_PerformancePage.js

```diff
@@ -1057,19 +1057,22 @@ class CDUPerformancePage {
             SimVar.SetSimVarValue("L:AIRLINER_ACC_ALT", "Number", accAlt);
         }
     }
-    static UpdateEngOutAccFromOrigin(mcdu) {
-        if (mcdu.engineOutAccelerationAltitudeIsPilotEntered) {
-            return;
-        }
+    static async UpdateEngOutAccFromOrigin(mcdu, updateEngOutAccAlt = true) {
```

Comment:
It doesn't look like we ever pass this in anywhere.
```suggestion
    static async UpdateEngOutAccFromOrigin(mcdu) {
```

Reply:
Screen shots were causing an issue with the text not showing up.